tests/src/test/scala/cats/tests/NonEmptySetSuite.scala

```diff
@@ -216,7 +216,7 @@ class NonEmptySetSuite extends CatsSuite {
 
   test("NonEmptySet#zipWithIndex is consistent with Set#zipWithIndex") {
     forAll { (nes: NonEmptySet[Int]) =>
-      assert(nes.zipWithIndex.toSortedSet === (nes.toSortedSet.zipWithIndex))
+      assert(nes.toSortedSet.zipWithIndex === nes.zipWithIndex.toSortedSet)
```

Comment:
Why is the equality flipped?

Reply:
I'ts because `===` in ScalaTest doesn't require type equality, whereas cats' `===` does. So you can't do `x: SortedSet[A] === y: Set[A]`, but  `y: Set[A] === x: SortedSet[A]` will compile just fine. And given the symmetry law for `===` I wager this is fine.